integrations/playwright-reporter.mdx

```diff
@@ -0,0 +1,229 @@
+---
+title: 'Playwright Reporter'
+description: 'Learn how to use the Checkly Playwright Reporter.'
+sidebarTitle: 'Playwright Reporter'
+---
+
+<Info>
+  **The Playwright Reporter is currently in a public beta**. [Contact the Checkly team](https://app.checklyhq.com/?support=true) for support or feedback. Expect breaking changes, features in development, and possible bugs.
+</Info>
+
+![CLI interface in front of the Playwright test session report](/images/docs/images/integrations/playwright-reporter.jpg)
+
+Use the Playwright Reporter to seamlessly integrate Playwright test results with Checkly monitoring. Automatically upload test results, screenshots, videos, and traces to gain visibility into your application's health across test runs.
+
+<Accordion title="Prerequisites">
+Before using the Playwright reporter, ensure you:
+
+- use Node.js >= 18.0.0
+- use Playwright >= 1.40.0
+- have Checkly account ([sign up](https://www.checklyhq.com/signup))
+</Accordion>
+
+
+## Install the Playwright Reporter
+
+Install [the Playwright reporter package](https://www.npmjs.com/package/@checkly/playwright-reporter) via npm.
+
+```bash
+npm install --save-dev @checkly/playwright-reporter
+```
+
+## How to get started
+
+### Create an API key and get your Account ID
+
+To start pushing your Playwright test reports to Checkly you need to authenticate the reporter with your Checkly Account ID and an API Key.
+
+1. [Log in to Checkly](https://app.checklyhq.com)
+2. Navigate to [your account Settings](https://app.checklyhq.com/settings/account/general) to get your account id (Account Settings > General)
+3. Create a new API key in [your user settings](https://app.checklyhq.com/settings/user/api-keys) (User Settings > Api Keys)
+
+Define your account id and API key as environment variables.
+
+```bash
+export CHECKLY_API_KEY=cu_123....
+export CHECKLY_ACCOUNT_ID=b2f...
+```
+
+<Tip>The Playwright reporter will automatically pick up the environment variables `CHECKLY_API_KEY` and `CHECKLY_ACCOUNT_ID`.</Tip>
```

Comment:
Given that this is only for the given terminal session, do we want them to have this configured in their env? or similar?

Reply:
Let's fix this then. More beginner friendly is always better. :)